packages/wrangler/e2e/versions.test.ts

```diff
@@ -590,13 +590,13 @@ describe("versions deploy", { timeout: TIMEOUT }, () => {
 	});
 
 	// TODO: revisit once AUS stabilises/works
-	it.skip("currently fails to upload if using experimental assets", async () => {
+	it.skip("currently fails to upload if using assets", async () => {
```

Comment:
This can maybe be unskipped now?

Not related to this PR though, so not at all blocking.

Reply:
Following up on  that one here - https://github.com/cloudflare/workers-sdk/pull/6803

Reply:
@emily-shen I've already addressed this in this PR

Reply:
will rebase on that once its in then :)